src/store/creator/reducer.ts

```diff
@@ -30,16 +31,12 @@ export default createReducer<CreatorState>(initialState, builder =>
             state.error = error;
             state.status = CreatorStatus.rejected;
         })
-        .addCase(resetCreatorAction, state => {
-            state.message = '';
-            state.staged = null;
-            state.status = CreatorStatus.init;
-        })
+        .addCase(resetCreatorAction, () => initialState)
```

Comment:
Is this change needed as part of the fix? It looks like it resets many more properties than before.

Reply:
It is in the sense that I wanted to reset the `referenceId` as well, the other properties (`cursor`, `error`) seemed appropriate to reset as well, but I can certainly just explicitly reset `referenceId` if there is concern

Reply:
I would err on the side of caution to keep the risk profile as low as possible.